<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #D0D0D0;</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Montserrat';
font-style: normal;
font-weight: 700;
font-size: 45px;
line-height: 122px;
color: #000000;</string>
   </property>
   <property name="text">
    <string>Ведомость</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="exams_list">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>331</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView::item
{
margin: 5px;
height: 30px;
color: #000;
background-color:#FFF;
border-radius:15px;
padding:0 10px;
}
QListView{
border: none;
}
QListView::item:selected
{
border:none;
}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>245</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	 background-color: #62B1FF;
	border-radius: 20%;
	font-family: 'Montserrat';
	font-style: normal;
	font-weight: 400;
	font-size: 15px;
	color: #000000;
}
QPushButton:hover{
   background-color:#7DBFFF;
}



</string>
   </property>
   <property name="text">
    <string>Добавить экзамен</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
